src/librustdoc/clean/types.rs

```diff
@@ -649,9 +649,8 @@ impl Item {
                         hir::Safety::Unsafe
                     },
                     abi,
-                    constness: if abi == Abi::RustIntrinsic
-                        && tcx.is_const_fn(def_id)
-                        && is_unstable_const_fn(tcx, def_id).is_none()
```

Comment:
I'm not sure what this intrinsic special-casing was about, so I should probably look into if we still need it.

Reply:
Ok, [it seems](https://github.com/rust-lang/rust/pull/106751#discussion_r1067957718) like it was just to be conservative, but I can't think of any reason why `tcx.is_const_fn` would "lie" to us here. Is there any reason this needs to be kept? cc @clubby789 @GuillaumeGomez

Reply:
Remove it and see what's breaking? :laughing: